src/features/ui/Wrapper/Wrapper.tsx

```diff
@@ -0,0 +1,30 @@
+import { FC } from 'react';
+
+import { InfoTooltip } from '@zero-tech/zui/components/InfoTooltip';
+
+import styles from './Wrapper.module.scss';
+import classNames from 'classnames';
+
+export type WrapperProps = {
+	children: React.ReactNode;
+	className?: string;
+	label: string;
+	info: string;
+};
+
+export const Wrapper: FC<WrapperProps> = ({
```

Comment:
At first look, I'm not 100% sure what `Wrapper` is used for. Would there be any alternatives to naming here?

Reply:
Definitely agree! Have changed to `InputWrapper` to give a bit more context. Let me know if you think this fits the bill.